content/docs/start/model-management/model-cicd.md

```diff
@@ -0,0 +1,217 @@
+---
+title: 'Get Started: Using and deploying models'
+description:
+  'Easily download your models from the model registry. Set up your CICD
+  pipelines to be trigger by model registry actions (such as assigning model
+  stages) and deploy models directly form the model registry.'
+---
+
+# Get Started: Using and Deploying Models
+
+In this chapter, we will learn how to access and use models and how to use the
+model registry to trigger automated CICD model workflows.
+
+## Pushing model artifacts to the remote
+
+If you are following along from the
+[model registry chapter](/doc/start/model-management/model-registry), you have
+trained a model and registered it to the model registry. However, to start using
+and sharing the model artifacts we need to push them to our DVC remote.
+
+Provided that you have the
+[DVC remote set up](/doc/start/data-management/data-versioning#configuring-a-remote),
+you now just need to call
+
+```console
+dvc push
+```
+
+from our project's git repository. All data and artifacts (such as models) which
+we created and which are versioned by DVC will now be uploaded from our local
+DVC cache to the DVC Remote.
+
+<admon type="tip">
+
+To learn about how the DVC Remote works and how to set it up, have a look at the
+[Get Started guide on Data management](/doc/start/data-management/data-versioning)
+or the [Remote Storage Guide](/doc/user-guide/data-management/remote-storage)
+for even more detail.
+
+</admon>
+
+## Downloading models
```

Comment:
Instead of starting with `dvc artifacts get`, since we are already using Studio, why not start with that method? I think `dvc artifacts get` and other methods might even be able to be hidden in details section like you do for gto on the previous page.

Reply:
Yes, that makes a lot of sense

Reply:
Hmm, but this is not going to work without pushing to the new remote

Reply:
Oh wait, that's not true, the remote is public and the forked repo can still read from it, so it should work fine.